lib/browser/api/menu.js

```diff
@@ -133,7 +133,7 @@ Menu.sendActionToFirstResponder = bindings.sendActionToFirstResponder
 
 // set application menu with a preexisting menu
 Menu.setApplicationMenu = function (menu) {
-  if (!(menu || menu.constructor === Menu)) {
+  if (menu && menu.constructor !== Menu)) {
```

Comment:
There are one to many closing brackets here 😄

Reply:
That's what you get for trying to fix this on the phone 🗡